clinica/pipelines/statistics_volume/statistics_volume_utils.py

```diff
@@ -0,0 +1,497 @@
+    for old_name, new_name in zip(figures, new_figure_names):
+        copyfile(old_name, new_name)
+
+    # Handle spm t maps
+    spm_T = [abspath(join(dirname(spm_mat), f)) for f in list_files if f.startswith('spmT')]
+    if len(spm_T) != 2:
+        raise RuntimeError('[Error] ' + str(len(spm_T)) + ' SPM t-map(s) were found')
+    if fwhm:
+        spmT_0001 = abspath('group-' + group_id + '_' + class_names[0] + '-lt-' + class_names[1] + '_measure-' + measure + '_fwhm-' + str(int(fwhm)) + '_TStatistics.nii')
+        spmT_0002 = abspath('group-' + group_id + '_' + class_names[1] + '-lt-' + class_names[0] + '_measure-' + measure + '_fwhm-' + str(int(fwhm)) + '_TStatistics.nii')
+    else:
+        spmT_0001 = abspath('group-' + group_id + '_' + class_names[0] + '-lt-' + class_names[1] + '_measure-' + measure + '_TStatistics.nii')
+        spmT_0002 = abspath('group-' + group_id + '_' + class_names[1] + '-lt-' + class_names[0] + '_measure-' + measure + '_TStatistics.nii')
+    copyfile(join(dirname(spm_mat), 'spmT_0001.nii'), spmT_0001)
+    copyfile(join(dirname(spm_mat), 'spmT_0002.nii'), spmT_0002)
+
+    variance_of_error = abspath('./group-' + group_id + '_VarianceError.nii')
+    copyfile(abspath(join(dirname(spm_mat), 'ResMS.nii')), variance_of_error)
+
+    resels_per_voxels = abspath('./resels_per_voxel.nii')
+    copyfile(abspath(join(dirname(spm_mat), 'RPV.nii')), resels_per_voxels)
+
+    mask = abspath('./included_voxel_mask.nii')
+    copyfile(abspath(join(dirname(spm_mat), 'mask.nii')), mask)
+
+    # Handle beta files
+    betas = [abspath(join(dirname(spm_mat), f)) for f in list_files if f.startswith('beta_')]
+    if len(betas) != 2 + len(covariables):
+        raise RuntimeError('[Error] Not enough betas files found in output directory')
+    regression_coeff_covar = [abspath('./' + covar + '.nii') for covar in covariables]
+    regression_coeff = [abspath('./' + class_names[0] + '.nii'),
+                        abspath('./' + class_names[1] + '.nii')]
+    regression_coeff.extend(regression_coeff_covar)
+    # Order is respected:
+    for beta, reg_coeff in zip(betas, regression_coeff):
+        copyfile(beta, reg_coeff)
+
+    # Handle contrast files
+    con_files = [abspath(join(dirname(spm_mat), f)) for f in list_files if f.startswith('con_')]
+    if len(con_files) != 2:
+        raise RuntimeError('There must exists only 2 contrast files !')
+    contrasts = [abspath('group-' + group_id + '_' + class_names[0] + '-lt-' + class_names[1] + '_measure-' + measure + '_contrast.nii'),
+                 abspath('group-' + group_id + '_' + class_names[1] + '-lt-' + class_names[0] + '_measure-' + measure + '_contrast.nii')]
+    for con, contrast in zip(con_files, contrasts):
+        copyfile(con, contrast)
+
+    return spmT_0001, spmT_0002, new_figure_names, variance_of_error, resels_per_voxels, mask, regression_coeff, contrasts
+
+
+def add_spm2txt(script_file, matlab_path):
```

Comment:
Missing docstring.

Reply:
Function removed because not used in the code